common/vm_wait_primary_nic_mac.yml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021-2023 VMware, Inc.
```

Comment:
Copyright should only have 2023. And please add a description about this task file into `common/README.md`.

Reply:
change it. Thanks

Reply:
Copyright is still not correct.

Reply:
Done. Thanks.